client/clparser/argtable3.h

```diff
@@ -0,0 +1,305 @@
+/*******************************************************************************
```

Comment:
Can you please also include in `client/clparser/` a `README` file indicating the source of the library (ie: web page where you can download it from), what version it is, any modifications you've made to the library, and its license?